centos-quarkus-maven/Dockerfile

```diff
@@ -5,34 +5,37 @@ FROM centos:7
 
 ARG GRAAL_VERSION=1.0.0-rc14
 ARG MAVEN_VERSION=3.6.0
-
-ENV GRAAL_CE_URL=https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-${GRAAL_VERSION}-linux-amd64.tar.gz
+ARG USER_HOME_DIR="/root"
+ARG GRAAL_CE_URL=https://github.com/oracle/graal/releases/download/vm-${GRAAL_VERSION}/graalvm-ce-${GRAAL_VERSION}-linux-amd64.tar.gz
```

Comment:
Why as ARG instead of ENV?

Reply:
`ENV` are persisted with the final image, ARG are only used during build. in this case the ARG is better than ENV as we are not going to install GRAAL after the image is built.